index.html

```diff
@@ -23,6 +24,14 @@
           </ul>
         </nav>
       </header>
+     <div id="menu-option">
+        <i id="x-mark" class="fa-solid fa-x"></i>
+        <ul id="menu-list">
+            <li><a id="port" href="#recent">Portfolio</a></li>
+            <li><a id="about-link" href="#about-div">About</a></li>
+            <li><a id="contact-link" href="#contact-block">Contact</a></li>
+        </ul>
+    </div>
```

Comment:
- [ ] I would implore you to keep your code DRY by not duplicating the navigation. You can use the one you used for the desktop version, all you need to do is to use the CSS media query to control the visibility and the layout on the respective view. Your app works fine but it is not professional to have two navigations working on the same app. If this is done, then the closing function in the script file would not be needed.